staging/src/k8s.io/pod-security-admission/admission/api/types.go

```diff
@@ -42,3 +46,47 @@ type PodSecurityExemptions struct {
 	Namespaces     []string
 	RuntimeClasses []string
 }
+
+
+// Attributes exposes the admission request parameters consumed by the PodSecurity admission controller.
+type Attributes interface {
```

Comment:
I think this should be in `pod-security-admission/api` not `pod-security-admission/admission/api`.

We probably need to reorganize those directories, but the first is the internal one.